validator/src/commands/run/args.rs

```diff
@@ -1798,6 +1824,92 @@ mod tests {
         );
     }
 
+    #[test]
+    fn verify_args_struct_by_command_run_with_ledger_path() {
+        // unexisting absolute ledger path
+        {
+            let default_run_args = RunArgs::default();
+            let tmp_dir = fs::canonicalize(tempfile::tempdir().unwrap()).unwrap();
+            let ledger_path = tmp_dir.join("my_unexisting_ledger_path");
```

Comment:
nit: Can probably drop `my` from the name here and below

Reply:
sure! [1ee75ac](https://github.com/anza-xyz/agave/pull/8085/commits/1ee75acc7064f305309d0a7a5c659f9472c49ec2)